src/mdlComponentHandler.js

```diff
@@ -295,7 +294,7 @@ componentHandler = (function() {
     });
 
     if (config.constructor.prototype
-        .hasOwnProperty(componentConfigProperty_)) {
+            .hasOwnProperty(componentConfigProperty_)) {
```

Comment:
Yea, something auto-touched this in PHPStorm. I'll fix up the indenting real quick manually.